src/PluginsSystem/Microsoft.Performance.Toolkit.Plugins.Runtime/Loading/FileSystemInstalledPluginLoader.cs

```diff
@@ -65,10 +79,21 @@ public async Task<bool> LoadPluginAsync(InstalledPlugin installedPlugin)
 
             if (!await this.installedPluginValidator.ValidateInstalledPluginAsync(installedPlugin.Info))
             {
-                this.logger.Error($"Plugin {pluginIdentity} is not valid. It will not be loaded.");
+                this.logger.Error($"Plugin {pluginIdentity} is not valid installed plugin. It will not be loaded. Has it been corrupted?");
```

Comment:
>Has it been corrupted? [](http://example.com/codeflow?start=114&length=22)

Should we add some hash information so that we can determine whether it's been corrupted? Seems like a strange question to ask in a log comment.

Reply:
I updated the implementation to log that message